README.md

```diff
@@ -1 +1,8 @@
-# PythonHW_Iterator_Generator
\ No newline at end of file
+### PythonHW_Iterator_Generator
+
+#### Contains Python 3 code for classes IteratorEnhancedRange and GeneratorEnhancedRange.
```

Comment:
Why we use headers all the way here?

Reply:
I thought it looked better that way. Changed to normal font.